### ISSUE TITLE
SCREEN-VALUE attribute not recognized as a keyword

### ISSUE THREAD

**Issue by gquerret** [Riverside-Software/sonar-openedge#651]
Reported by MF

And thus can't be used in the MatchingKeyword template rule.

**Comment by gquerret** [comment]
Also those ones:
* SCREEN-VALUE
* CHECKED
* SENSITIVE
* HIDDEN
* VISIBLE

**Comment by gquerret** [comment]
Merged in develop, will be available in next release

**Comment by mikefechner** [comment]
I'm getting a bunch of those messages in the Jenkins job reporting to SonarCube:

```
15:49:04 [sonar:sonar] Unable to find token number for keyword SCREEN-VALUE
15:49:04 [sonar:sonar] Unable to find token number for keyword SCREEN-VALUE
15:49:04 [sonar:sonar] Unable to find token number for keyword SCREEN-VALUE
15:49:04 [sonar:sonar] Unable to find token number for keyword SCREEN-VALUE
15:49:04 [sonar:sonar] Unable to find token number for keyword SCREEN-VALUE
15:49:04 [sonar:sonar] Unable to find token number for keyword SCREEN-VALUE
15:49:05 [sonar:sonar] Unable to find token number for keyword SCREEN-VALUE
15:49:05 [sonar:sonar] Unable to find token number for keyword SCREEN-VALUE
15:49:05 [sonar:sonar] Unable to find token number for keyword SCREEN-VALUE
```

Is that related?

**Comment by gquerret** [comment]
Yes, this was included in 2.4.0.
